src/doc/src/reference/unstable.md

```diff
@@ -508,14 +508,27 @@ CLI paths are relative to the current working directory.
 * Original Pull Request: [#9322](https://github.com/rust-lang/cargo/pull/9322)
 * Tracking Issue: [#9453](https://github.com/rust-lang/cargo/issues/9453)
 
-The `target-applies-to-host` key in a config file can be used set the desired
-behavior for passing target config flags to build scripts.
-
-It requires the `-Ztarget-applies-to-host` command-line option.
-
-The current default for `target-applies-to-host` is `true`, which will be
-changed to `false` in the future, if `-Zhost-config` is used the new `false`
-default will be set for `target-applies-to-host`.
+Historically, Cargo has respected the `linker` and `runner` options (but
+_not_ `rustflags`) of `[target.<host triple>]` configuration sections
+when building and running build scripts, plugins, and other artifacts
+that are _always_ built for the host platform.
```

Comment:
I don't think this is quite right? The `runner` option isn't used at all for build scripts (ever I think?). Additionally I believe if you type `cargo build` then Cargo does indeed apply both RUSTFLAGS and `target.$triple.rustflags` to build scripts (much to lots of folks' chagrin). I believe this is accurate (I think?) when `cargo build --target $host` is executed, though. (in that `target.$host.rustflags` is ignored but `target.$host.linker` is probably respected)

Reply:
Yes, you're exactly right. I've updated the text.